.github/workflows/publish-backend-plugin-manager.yaml

```diff
@@ -0,0 +1,173 @@
+# Copyright 2023 Janus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Publish Backend Plugin Manager
+
+on:
+  push:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+env:
+  SCOPE: ${{ vars.BACKEND_PLUGIN_MANAGER_SCOPE }}
+
+jobs:
+  build:
+    # build the backend plugin manager in the backstage/backstage GitHub repository
+    # and publish it to the NPMJS repository @janus-idp organization,
+    # with changed scope and version
+    name: Publish Backend Plugin Manager
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          repository: backstage/backstage
+          ref: master
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
```

Comment:
```suggestion
          node-version-file: '.nvmrc'
          cache: 'yarn'
```

Reply:
Good call on `.nvmrc`. 👍 Can we still enable the cache?

Reply:
I added `cache: 'yarn'` as you requested.